src/main/client/type.c

```diff
@@ -332,6 +332,9 @@ static PyMethodDef AerospikeClient_Type_Methods[] = {
 
     // ADMIN OPERATIONS
 
+    {"admin_create_pki_user",
+     (PyCFunction)AerospikeClient_Admin_Create_PKI_User,
+     METH_VARARGS | METH_KEYWORDS, "Create a new user."},
```

Comment:
```suggestion
     METH_VARARGS | METH_KEYWORDS, "Create a new pki user."},
```
nit: A little extra to differentiate it from "admin_create_user."